### PR TITLE
bpo-20104: Change the file_actions parameter of os.posix_spawn().

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
* Make its default value an empty tuple instead of None.
* Make it a keyword-only parameter.

https://bugs.python.org/issue20104

### CODE REVIEW

**Review by pablogsal — APPROVED** [review]
LGTM Do you want to wait until #6693 is merged to land this or you prefer to do it before?

**Comment by serhiy-storchaka** [comment]
It doesn't mean what PR will be merged first. But I want to wait for a review from any of other involved core devs: @benjaminp or @gpshead.

**Pull request merged**
